x-pack/plugins/apm/public/services/rest/apm/services.ts

```diff
@@ -50,23 +49,3 @@ export async function loadServiceDetails({
     }
   });
 }
-
-export async function loadServiceEnvironments({
-  serviceName,
-  start,
-  end
-}: {
-  serviceName?: string;
-  start: string;
-  end: string;
-}) {
-  const pathname = `/api/apm/services/environments`;
-  return callApi<ServiceEnvironmentsAPIResponse>({
-    pathname,
-    query: {
-      start,
-      end,
-      serviceName
-    }
-  });
-}
```

Comment:
Move uiFilters methods to dedicated file (we'll have many more uiFilters coming soon and it'll be nice to have them co-located).